prog/mutation.go

```diff
@@ -156,7 +156,7 @@ func (ctx *mutator) mutateArg() bool {
 	if len(p.Calls) == 0 {
 		return false
 	}
-	c := p.Calls[r.Intn(len(p.Calls))]
+	c := ctx.chooseCall()
```

Comment:
I think it makes sense to pass `p` here as an argument.

Reply:
Fixed. And I also added the _r_ argument used by the _randomChoice_ function.